R/bootstrap-model.R

```diff
@@ -86,15 +86,8 @@ new_bootstrap_run <- function(
 #' @param strat_cols columns to maintain proportion for stratification
 #' @param seed a seed for sampling the data. Set to `NULL` to avoid setting.
 #' @param data A dataset to resample from. Defaults to `NULL`, which will use
```

Comment:
Maybe check the help topic for consistency in capitalization for first letter in arg description.

Reply:
Yeah this has been a `bbr` issue for a while now. We seem to go back and forth across a lot of functions and dont really seem to have a standard. Ill update the bootstrap ones in this PR, but there are a couple inherited from other functions (like `delete_models` in `cleanup_bootstrap_run`) that I feel like would be out of scope for this PR